htdocs/product/card.php

```diff
@@ -201,7 +201,7 @@
     {
         $error = 0;
 
-        if (!GETPOST('label', 'alphanohtml'))
```

Comment:
You use HTML inside a short label ?

Reply:
>You use HTML inside a short label ? This introduce a lot of security problem.
I agree it should not be encouraged; I modified the PR so that the default will be `'alphanohtml'`.